node/impl/client/client_test.go

```diff
@@ -44,6 +46,7 @@ func TestImportLocal(t *testing.T) {
 	b, err := testdata.ReadFile("testdata/payload.txt")
 	require.NoError(t, err)
 
+	//stm @CLIENT_STORAGE_DEALS_LIST_IMPORTS_001
```

Comment:
Why do some lines use `//stm:`, and others `//stm`?

Reply:
The correct form is `//stm:`. The other lines are typos, and will be fixed.